python/pyspark/sql/functions.py

```diff
@@ -377,6 +377,23 @@ def cosh(col):
     return _invoke_function_over_column("cosh", col)
 
 
+def cot(col):
```

Comment:
actually you should also add them in functions.R too. ref: https://github.com/apache/spark/commit/aeb45df668a97a2d48cfd4079ed62601390979ba

Reply:
Sorry, I forgot about that. I'm pushing the follow-up PR soon.

Reply:
https://github.com/apache/spark/pull/33925

Reply:
this is fine. usually SparkR one is done separately in a separate JIRA.